web/src/installers/versions.js

```diff
@@ -97,6 +97,7 @@ module.exports.InstallerVersions = {
   ],
   weave: [
     // cron-weave-update-265
+    "2.8.1-20221122",
```

Comment:
This looks wrong

Reply:
Why? That was added by the script automatically ./generate.sh
It follows the same approach of : https://github.com/replicatedhq/kURL/pull/3628/files

Reply:
Because it's 281 in the 265 section, and it's in here twice (shall on line 110)